src/main/java/frc/robot/Robot.java

```diff
@@ -279,28 +305,47 @@ public void teleopSetup() {
     elevator.start();
     shooter.start();
     ballSubsystem.start();
-  }
 
+
+  }
+  TalonSRX turretMotor = new TalonSRX(RobotMap.TURRET_PORT);
```

Comment:
These should be declared at the top of the file with everything else, and initialized in `robotSetup`